src/controllers/controller.line.js

```diff
@@ -5,23 +5,14 @@ import {resolve} from '../helpers/helpers.options';
 
 export default class LineController extends DatasetController {
 
-	constructor(chart, datasetIndex) {
-		super(chart, datasetIndex);
-
-		this._showLine = false;
-	}
-
 	update(mode) {
 		const me = this;
 		const meta = me._cachedMeta;
 		const line = meta.dataset;
 		const points = meta.data || [];
-		const options = me.chart.options;
-		const config = me._config;
-		const showLine = me._showLine = valueOrDefault(config.showLine, options.showLines);
 
 		// Update Line
-		if (showLine && mode !== 'resize') {
+		if (mode !== 'resize') {
```

Comment:
it could be worth a comment here explaining why we skip this when mode is resize